src/consolebundle/CommandRunner.py

```diff
@@ -20,13 +26,15 @@ def run_command():
     logger = container.get("consolebundle.logger")
     logger.warning("Running command in {} environment".format(known_args.env.upper()))
 
-    if len(sys.argv) < 2:
+    if len(sys.argv) < 2 or (known_args.help_selected and len(known_args.command_name) < 0):
         logger.error("Command not specified, example usage: console mynamespace:mycommand")
 
-        print("\n[Available commands]:")
+        command_manager.log_commands(logger)
+
+        sys.exit(1)
+    elif command_manager.command_prefix_only(known_args.command_name):
 
-        for existing_command in command_manager.get_commands():
-            logger.info(existing_command.get_command() + " - " + existing_command.get_description())
+        command_manager.log_subcommands(known_args.command_name, logger)
 
         sys.exit(1)
```

Comment:
Why exist(1) ? It is not an error, isn't it?

Reply:
Yup, sorry, this was a copy paste.